ks-component-ts/src/field/components/PriceInput/PriceInputControlled/helpers.ts

```diff
@@ -10,7 +10,13 @@ export const onValueChange = (
 ): void => {
   onChange(
     event.floatValue && event.floatValue > 0
-      ? (event.floatValue as unknown as number) * currencyDecimal
+      ? parseInt(
```

Comment:
Not sure on `parseInt` with `toPrecision` - floating points will always be e.g. `9.99997` rather than `10`, so I think just `Math.ceil()` might be better. WDYT?